packages/web/src/scss/components/Modal/README.md

```diff
@@ -369,3 +384,5 @@ Or feel free to write controlling scripts yourself.
 [scroll-view]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/ScrollView/README.md
 [web-readme]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/README.md
 [dictionary-alignment]: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/DICTIONARIES.md#alignment
+[readme-deprecations]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web/README.md#deprecations
```

Comment:
Not sure who it was, but someone told me that we are mixing blob/tree links.

Reply:
I checked [what's behind it](https://stackoverflow.com/questions/39400848/in-github-urls-what-is-the-difference-between-a-tree-and-a-blob) and both are correct, each for different use case:

- **blob** is a Git object for file,
- **tree** is for directory.

Which means you are right, `blob` should be here instead of `tree`. 👍🏻